README.md

```diff
@@ -1,878 +1,21 @@
-# @kinde-oss/kinde-typescript-sdk 
-The `@kinde-oss/kinde-typescript-sdk` allows developers to integrate [**Kinde**](https://kinde.com)
-Authentication into their TypeScript projects. This SDK implements the following
-OAuth2.0 flows.
+# Kinde TypeScript SDK
 
-- **Authorization Code**    
-  Intended for confidentials clients for e.g. web-servers
+The TypeScript SDK for Python.
```

Comment:
@oliwolff1 which SDK?

Reply:
@onderay have a look at the latest PR, I quickly updated the PR after I submitted it as I noticed this.